lib/elixir_bench/github/client_http.ex

```diff
@@ -0,0 +1,50 @@
+defmodule ElixirBench.Github.ClientHTTP do
+  @moduledoc """
+  The Github HTTP client.
+
+  This client uses HTTP requests to gather data from yml files in given
+  repository hosted on Github. It uses the `:hackney` library to actually send
+  the requests and the `yamerl` library to decode the data in the yml format.
+
+  Requests follow a base url:
+
+    - base_url = 'https://raw.githubusercontent.com'
+
+  This module implements the following functions:
+
+    - a `get_yaml/1` function that takes the url path of the project and return
+      the raw parsed data in the yaml format.
```

Comment:
The information which methods are implemented isn't needed - `mix docs` generates it and otherwise you can see it looking at the module itself.